benchmark.sh

```diff
@@ -7,35 +7,43 @@ DURATION=120
 [ "$(uname -s)" == "Darwin" ] && SED=gsed
 [ "$(uname -s)" == "Darwin" ] && HEAD=ghead
 
+BENCHMARK_NAME="$1"
+if [ -f "${BENCHMARK_NAME}.yml" ]; then
+  DOCKER_COMPOSE_YML="${BENCHMARK_NAME}.yml"
+elif [ -f "${BENCHMARK_NAME}/docker-compose.yml" ]; then
+  DOCKER_COMPOSE_YML="${BENCHMARK_NAME}/docker-compose.yml"
+else
+  DOCKER_COMPOSE_YML=""
+fi
+
 # if we're done, clean up all child processes
-if [ -z "$1" ] || [ ! -f "$1.yml" ]; then
+if [ -z "${DOCKER_COMPOSE_YML}" ] ; then
   echo Usage:
   echo
   echo "  $0 testsuite"
-  echo 
+  echo
   echo Available testsuites:
   echo
-  find -- * -name \*.yml | $SED -e 's/\.yml$//' | sort
+  ls */docker-compose.yml | $SED -e 's,/docker-compose.yml$,,' | sort | uniq
```

Comment:
Does a file system guarantee the uniqueness of these values? ( can we trim `uniq`? )

Reply:
Sorry. This is needless.
It's needed if we keep backward compatibility.

I'll remove this too.